src/layers/domain/core/questionnaire.py

```diff
@@ -27,49 +30,238 @@ class Question(BaseModel):
     name: str
     type: QuestionType
     multiple: bool
+    validation_rules: list[str] = []
+    choices: list[str] = []
```

Comment:
change to set as we don't want duplicates and set will offer us some nicer performance